### ISSUE TITLE
The status bar can be set to not display the VPN indicator when VPN is enabled; DivestOS is often experiencing automatic pauses during music playback, even for downloaded music;Please support scrolling screenshot

### ISSUE THREAD

**Issue by Subtly7360** [Divested-Mobile/DivestOS-Build#216]
- The status bar can be set to not display the VPN indicator when VPN is enabled.
I did not see the VPN option in the status bar,keep the VPN turned on in the drop down notification shade.
Especially when turning on a VPN, it is important for people in authoritarianism/autocracy/despotism countries to be able to adjust the status bar to not display the VPN usage indicator.

- even though it's already the latest version of DivestOS, why is there still a prompt to download the latest Rom package? 

- Please support scrolling screenshot.
- I often experience automatic pauses while listening to music on Spotify, and I downloaded Spotify from Aurora Store.

**Comment by SkewedZeppelin** [comment]
> The status bar can be set to not display the VPN indicator when VPN is enabled;

Strange it indeed doesn't show as an option in Settings > System > Status bar > System icons.

> why is there still a prompt to download the latest Rom package

Feature not a bug

> A screen recording software that can record and capture browser operations is needed.

There is Recorder built in, and some others available on F-Droid.

**Comment by Subtly7360** [comment]
> Strange it indeed doesn't show as an option in Settings > System > Status bar > System icons.

No VPN.

keep the VPN turned on in the drop down notification shade.

> There is screen recorder built in you have to add the quick settings tile for it, and some others available on F-Droid.

Is there a built-in screen recorder? Where is it? How do I bring it out to use it?

**Comment by SkewedZeppelin** [comment]
> Is there a built-in screen recorder?

Pull down the quick settings all the way, tap the pencil/edit, add the `Screen record` tile.

**Comment by Subtly7360** [comment]
> Strange it indeed doesn't show as an option in Settings > System > Status bar > System icons.

No VPN.

keep the VPN turned on in the drop down notification shade.

> There is screen recorder built in you have to add the quick settings tile for it, and some others available on F-Droid.

Is there a built-in screen recorder? Where is it? How do I bring it out to use it?

**Comment by Subtly7360** [comment]
> > Is there a built-in screen recorder?
> 
> Pull down the quick settings all the way, tap the pencil/edit, add the `Screen record` tile.

- The status bar can be set to not display the VPN indicator when VPN is enabled; 
I did not see the VPN option in the status bar,keep the VPN turned on in the drop down notification shade.
Especially when turning on a VPN, it is important for people in authoritarianism/autocracy/despotism countries to be able to adjust the status bar to not display the VPN usage indicator.

- even though it's already the latest version of DivestOS, why is there still a prompt to download the latest Rom package? 

- Please support scrolling screenshot.
- I often experience automatic pauses while listening to music on Spotify, and I downloaded Spotify from AuroraStore.

**Comment by SkewedZeppelin** [comment]
> why is there still a prompt to download the latest Rom package?

This is a feature, not a bug.

>  Please support scrolling screenshot.

When you take a screenshot the bottom left has a double arrow button, tap it and it will scroll.

> I often experience automatic pauses while listening to music on Spotify

On Bluetooth or wired? If Bluetooth try disabling HD audio for your paired Bluetooth device. 

> VPN usage indicator.

This is very low on things I would change.

**Comment by Subtly7360** [comment]
> > why is there still a prompt to download the latest Rom package?
> 
> This is a feature, not a bug.
> 
> > Please support scrolling screenshot.
> 
> When you take a screenshot the bottom left has a double arrow button, tap it and it will scroll.
> 
> > I often experience automatic pauses while listening to music on Spotify
> 
> On Bluetooth or wired? If Bluetooth try disabling HD audio for your paired Bluetooth device.
> 
> > VPN usage indicator.
> 
> This is very low on things I would change.

I did not see a prompt button for taking long screenshots while taking a screenshot, and was only able to take partial screenshots.

not connected to Bluetooth and wired.

**Comment by SkewedZeppelin** [comment]
> I did not see a prompt button

https://www.xda-developers.com/how-to-take-scrolling-screenshots-android/#how-to-take-scrolling-screenshots-on-stock-android

> not connected to Bluetooth and wired.

Do you have the cutouts in any other audio apps?

**Comment by Subtly7360** [comment]
> Do you have the cutouts in any other audio apps?

No, I did not have any other apps open，The issue of automatic pausing while playing audio or music also exists in the Firefox Fast & Private Browser 114.2.0 downloaded from apkmirror.com, proving that there is a problem with DivestOS.

**Comment by Subtly7360** [comment]
> > I did not see a prompt button
> 
> https://www.xda-developers.com/how-to-take-scrolling-screenshots-android/#how-to-take-scrolling-screenshots-on-stock-android
> 
> > not connected to Bluetooth and wired.
> 
> Do you have the cutouts in any other audio apps?

DivestOS is often experiencing automatic pauses during music playback, even for downloaded music.
There is also plenty of free space left in the phone's memory, and no applications are running in the background.

**Comment by Subtly7360** [comment]
> * The status bar can be set to not display the VPN indicator when VPN is enabled.
>   I did not see the VPN option in the status bar,keep the VPN turned on in the drop down notification shade.
>   Especially when turning on a VPN, it is important for people in authoritarianism/autocracy/despotism countries to be able to adjust the status bar to not display the VPN usage indicator.
> * even though it's already the latest version of DivestOS, why is there still a prompt to download the latest Rom package?
> * Please support scrolling screenshot.
> * I often experience automatic pauses while listening to music on Spotify, and I downloaded Spotify from Aurora Store.
